app/webpacker/components/RegistrationsV2/Register/RegistrationClosedMessage.jsx

```diff
@@ -0,0 +1,48 @@
+import { DateTime } from 'luxon';
+import React, { useEffect, useState } from 'react';
+import { Message } from 'semantic-ui-react';
+import I18nHTMLTranslate from '../../I18nHTMLTranslate';
+
+export default function RegistrationClosedMessage({
+  registrationStart,
+  onTimerEnd,
+}) {
+  const [timeLeft, setTimeLeft] = useState({
+    days: 0, hours: 0, minutes: 0, seconds: 0,
+  });
+
+  const start = DateTime.fromISO(registrationStart);
+
+  // Update the timer every second
+  useEffect(() => {
+    const updateTimer = () => {
+      const now = DateTime.local();
+
+      if (now >= start) {
+        onTimerEnd();
+        return;
+      }
+
+      const diff = start.diff(now, ['days', 'hours', 'minutes', 'seconds']).toObject();
+      setTimeLeft({
+        days: Math.floor(diff.days),
+        hours: Math.floor(diff.hours),
+        minutes: Math.floor(diff.minutes),
+        seconds: Math.floor(diff.seconds),
+      });
+    };
+
+    const intervalId = setInterval(updateTimer, 1000); // Update every second
+    updateTimer(); // Initial call to set the state right away
+
+    return () => clearInterval(intervalId); // Cleanup interval on unmount
+  }, [registrationStart, onTimerEnd, start]);
+
+  return (
+    <Message color="blue">
+      { timeLeft.hours < 1
```

Comment:
```suggestion
      { timeLeft.days < 1 && timeLeft.hours < 1
```
otherwise 1 day 0 hours 1 minute 1 second would satisfy this.